### DISCUSSION TITLE
Outbound rtp video bitrate is very low

### DISCUSSION THREAD

**Discussion by farooq-zaman-82** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1711] [Q&A]
Hi,

My usecase involves running master in a browser using JavaScript SDK and viewer in an EC2 machine using C SDK and I observed that the outbound rtp video bitrate is very low, around 300 kpbs. Please see the attached image.

On the other hand when I run both master and viewer in browser using Javascript SDK bitrate is around 1.75 Mbps.

How can I achieve similar bit rate for my original usecase.

Any help would be highly appreciated.
<img width="324" alt="Screenshot 2023-05-10 at 7 17 10 PM" src="https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/assets/59040931/05aab54b-a62e-469b-a9b7-2ccf4a4c0b6d">

**Comment by disa6302** [comment]
@farooq-zaman-82 , 

Thank you for reaching out. What is the network scenario in case of the JS viewer and the C SDK viewer? Can you attach C SDK verbose logs? We need to understand what ICE candidates get selected for this process. 
We would also like to know what candidates get selected in the case where JS viewer is used.

**Reply by farooq-zaman-82** [reply]
Hi @disa6302 ,

Thank you for the response. Please find attached the logs for both master and viewer applications. Master is using JavaScript SDK while Viewer is using C SDK.

[master_javascript_sdk.txt](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/files/11529307/master_javascript_sdk.txt)
[viewer_c_sdk.txt](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/files/11529311/viewer_c_sdk.txt)

**Reply by farooq-zaman-82** [reply]
Hi @disa6302 ,

any update regarding the issue? BTW I am using GStreamer to capture video as HLS stream in C Application and video is one way only means C application receives video but doesn't send anything back to JS application.
